### PR TITLE
Bugfix/testfile location

### PR DESCRIPTION
**Pull request by fightingtexasaggie**
No more need to have the testfile.bin in a webserver aliased location since we have a true public folder, per instance.

### CODE REVIEW

File: public/js/download.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ function testDownload(fileSize) {
     let start = new Date().getTime();
     $.ajax({
         type: "GET",
-        url: "/download/testfile.bin?id=" + start,
+        "url": Routing.generate("pelagos_homepage") + '/images/testfile.bin?id=' + start,
```
Comment (praneethpr) [inline public/js/download.js]: Just to be consistent, use double quotes

File: public/js/download.js [hunk 2]
```diff
@@ -87,7 +87,7 @@ function testDownload(fileSize) {
     let start = new Date().getTime();
     $.ajax({
         type: "GET",
-        url: "/download/testfile.bin?id=" + start,
+        "url": Routing.generate("pelagos_homepage") + "/images/testfile.bin?id=" + start,
```
Comment (mickel1138) [inline public/js/download.js]: Just use asset('path to file')

**Comment by mickel1138** [comment]
Test file doesn't have to go into images folder, can be in the root, it's own folder, anywhere really.

File: public/js/download.js (hunk 2 above)
Comment (mickel1138) [reply public/js/download.js]: Ok, can't use asset tag in Javascript, so you'll have to set it somewhere in the template.

Comment (mickel1138) [reply public/js/download.js]: https://symfony.com/doc/4.3/frontend/encore/server-data.html

Comment (mickel1138) [reply public/js/download.js]: Ok, forget all that, I think it's easier if you just want to the base path, just get it from Routing like this: `Routing.getBaseUrl()`

**Pull request merged**